### ISSUE TITLE
Lyz citekey is different when exported and using lyxpipe

### ISSUE THREAD

**Issue by MaxBr** [willsALMANJ/lyz#17]
Hi,
I pushed all my citations to Lyx using the Lyxpipe from Lyz, and then updated the bibtex file and Lyx didn't find the correct citations anymore. When I use the export function from either Lyz of Zotero, I do not get the same format of the citekey that Lyz initially used.
Any comments ore workarounds on this are very welcome.
Best,
Max

**Comment by willsALMANJ** [comment]
I'm not sure what could have happened there. What settings do you have for LyZ? Can you give some examples of the citekeys before and after the change? Does LyZ consistently produce this new format now? If you try a new document with a new bib file and cite some of the same references, do you get the same format?

**Comment by MaxBr** [comment]
Thanks for your fast response.
I worked on this a bit more:
The bibtexkey when pushed by lyz is for example "hu2017insitu", authyear and the first two words of the title. When exporting from Zotero I can choose either shorttitle or veryshorttitle that come close to this. However, veryshorttitle gives "hu2017situ" and shorttitle gives "hu2017insitumolecular", because Zotero considers "in" as a filling word when using veryshorttitle.
It is now solved with a script that changes all citekeys in my lyxfile to authyear and exporting the same citekey format from Zotero/Jabref.
Thanks anyway.
